### ISSUE TITLE
Use softer background colors 

### ISSUE THREAD

**Issue by Buxuna** [SkyD666/AniVu#55]
I'm Dyslexic
For some people with dyslexia, high contrast color schemes can make reading/viewing content  more difficult and this is a very common disease . Currently I have noticed that the app uses really high contrasty backgrounds compared to other apps that I'm easily able to use like read you , I cant use the app in light mode because it feels so contrasty and in darkmode I still have problems because of a highcontrasty background , for comparison here is ReadYou background color comparison .
![Screenshot_2024-06-05-11-24-06-57_6de1e84f9daf512b0c3cf0ac2d20418d](https://github.com/SkyD666/AniVu/assets/171773426/ac3b67af-8553-4874-a17f-2c23d940c912) ![Screenshot_2024-06-05-11-42-08-55_8ddb5377535fa5c96d651cdcfe3fe939](https://github.com/SkyD666/AniVu/assets/171773426/5725ff44-e155-4e1a-a4bd-4a353b23376d)
![Screenshot_2024-06-05-11-42-35-22_6de1e84f9daf512b0c3cf0ac2d20418d](https://github.com/SkyD666/AniVu/assets/171773426/77cf1efe-cc92-4586-bde8-c57c7c571b08) ![Screenshot_2024-06-05-11-42-17-10_8ddb5377535fa5c96d651cdcfe3fe939](https://github.com/SkyD666/AniVu/assets/171773426/43326060-f866-40cf-949b-3671e4bab2f9)

I think if this very little consideration is taking into account many dyslexic people like me will be able to use the app with more ease . Please make the background colors more softer .

**Comment by SkyD666** [comment]
![image](https://github.com/SkyD666/AniVu/assets/32453770/4b1892af-ef10-4afa-be9f-fba11ad3f264)

Do you mean this contrast? This contrast is meant to identify that this article has been read

**Comment by SkyD666** [comment]
![Screenshot_20240605_151226_com.skyd.anivu.debug.jpg](https://github.com/SkyD666/AniVu/assets/32453770/dac22e18-84f8-4736-9b16-4129e7ca2a05)

Can you see this clearly?

**Comment by Buxuna** [comment]
Yes , it will be best to have a menu like read you, user can adjust the tonal elevation based on how much dyslexic they are

![Screenshot_2024-06-05-13-07-53-329_me.ash.reader-edit.jpg](https://github.com/SkyD666/AniVu/assets/171773426/03277218-a4c2-41b2-a409-501a6d698300)

![Screenshot_2024-06-05-13-07-29-134_me.ash.reader-edit.jpg](https://github.com/SkyD666/AniVu/assets/171773426/1072d747-932c-4b00-b70e-6d919f4f63e6)

![Screenshot_2024-06-05-13-06-55-191_me.ash.reader-edit.jpg](https://github.com/SkyD666/AniVu/assets/171773426/6b386ffe-61a1-482f-b20a-2d913150a5bb)

![Screenshot_2024-06-05-13-06-40-772_me.ash.reader-edit.jpg](https://github.com/SkyD666/AniVu/assets/171773426/bec9b589-16d7-45b6-b9eb-675c195fd0b1)

**Comment by Buxuna** [comment]
Thank you very much , this setting really helped me , for reading screen, i have no problems since I don't really read much articles.

**Comment by SkyD666** [comment]
https://github.com/SkyD666/AniVu/actions/runs/10796118349
